packages/console-types/src/types/catalog/well-known-items/sidecar/manifest.schema.json

```diff
@@ -269,6 +278,36 @@
                   "secretKey"
                 ],
                 "type": "object"
+              },
+              {
+                "additionalProperties": false,
+                "properties": {
+                  "description": {
+                    "type": "string"
+                  },
+                  "managedBy": {
+                    "type": "string"
+                  },
+                  "fieldPath": {
+                    "type": "string"
+                  },
+                  "name": {
+                    "minLength": 1,
+                    "type": "string"
+                  },
+                  "readOnly": {
+                    "type": "boolean"
+                  },
+                  "valueType": {
+                    "const": "downwardAPI"
+                  }
+                },
+                "required": [
+                  "fieldPath",
```

Comment:
aren't these required fields a breaking change?

Reply:
This is just a new possibile type of environment variable that wasn't (mistakingly) included before, it's an expansion of the interface